OloEngine/src/OloEngine/Networking/Lockstep/LockstepManager.cpp

```
@@ -0,0 +1,171 @@
+
+    u32 LockstepManager::GetCurrentTick() const
+    {
+        return m_CurrentTick;
+    }
+
+    void LockstepManager::SetHashCheckInterval(u32 interval)
+    {
+        m_HashCheckInterval = interval;
+    }
+
+    void LockstepManager::RecordStateHash(const std::vector<u8>& snapshotData)
```

Cppcheck (2.20.0)

[style] The function 'RecordStateHash' is never used.

(unusedFunction)